tests/basic/clone/clone.s

```diff
@@ -0,0 +1,25 @@
+.text
```

Comment:
Out of curiosity, why do you need assembly to invoke the clone syscall instead of using lkl_syscall directly?

Reply:
It needs to set up the stack explicitly.  The system call returns twice to the same instruction but with the new stack pointer.  You need to then pop the real call address from the stack somehow, which is easy in assembly and impossible to do in C without the compiler being allowed to break it.  Musl and glibc use assembly to call the clone system call directly (rather than via the `syscall` wrapper) for the same reason.